helm-chart/amalthea/templates/scheduler/rbac.yaml

```diff
@@ -37,9 +37,176 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: {{ include "amalthea.fullname" . }}-scheduler
 rules:
-  - apiGroups: [""]
-    resources: ["configmaps"]
-    verbs: ["get", "list", "watch"]
+  - apiGroups:
+    - ""
+    resources:
+    - configmaps
+    verbs:
+    - get
+    - list
+    - watch
+  # Copied from https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/main/jupyterhub/templates/scheduling/user-scheduler/rbac.yaml
+  - apiGroups:
```

Comment:
@aledegano do we really need all this? I wonder if some of this is JH specific that we do not need.

Reading at the docs on k8s seem to indicate that you can just make a clusterrole binding that adds

```
name: system:kube-scheduler
  apiGroup: rbac.authorization.k8s.io
```

I am wondering if you already tried this? If not I can try it on my deployment since I have this running. Everything else is good.

Reply:
This is the section of the docs I am referring to: https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#define-a-kubernetes-deployment-for-the-scheduler

Reply:
This is the section of the docs I am referring to: https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#define-a-kubernetes-deployment-for-the-scheduler

Reply:
ah, looking more carefully at my own copy-pasta: the role binding you are suggesting to add it's already there at L15 and unfortunately was not enough as the scheduler would complain of not having enough permissions.

Reply:
I'm reading the documentation a bit better and realized that the way to configure the scheduler has changed and what I've been using here is deprecated.
I'll close this PR while I go back to the drawing board.